module/zstd/Makefile.in

```diff
@@ -20,6 +20,10 @@ ccflags-y += -O3
 # Set it for other compilers, too.
 $(obj)/lib/zstd.o: c_flags += -fno-tree-vectorize
 
+# SSE register return with SSE disabled if -march=znverX is passed
+# https://github.com/openzfs/zfs/issues/10758
```

Comment:
I don't think we need to explicitly reference the issue here.  The git commit comment will include the issue number for future reference.  Would you mind removing the link.